app/src/main/java/com/example/dogbook/LoginActivity/LoginActivity.java

```diff
@@ -31,6 +31,11 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
 
+        //Persistent log in
+        if(ParseUser.getCurrentUser() != null) {
+            goMainActivity();
+        }
```

Comment:
I would return here to make it abundantly clear that the code that comes after this won't actually be executed